modules/cli/src/main/scala/scala/cli/exportCmd/MavenProjectDescription.scala

```diff
@@ -0,0 +1,236 @@
+import scala.cli.ScalaCli
+import scala.cli.exportCmd.POMBuilderHelper._
+import scala.xml.Elem
+
+object POMBuilderHelper {
+  def buildNode(name: String, value: String): Elem =
+    new Elem(
+      null,
+      name,
+      scala.xml.Null,
+      scala.xml.TopScope,
+      minimizeEmpty = false,
+      scala.xml.Text(value)
+    )
+}
+
+final case class MavenProjectDescription(extraSettings: Seq[String], logger: Logger)
+    extends ProjectDescriptor {
+  private val q  = "\""
+  private val nl = System.lineSeparator()
+
+  private def sources(sourcesMain: Sources, sourcesTest: Sources): MavenProject = {
+    val mainSources = ProjectDescriptor.sources(sourcesMain)
+    val testSources = ProjectDescriptor.sources(sourcesTest)
+    MavenProject(
+      mainSources = mainSources,
+      testSources = testSources
+    )
+  }
+
+  private def javaOptionsSettings(options: BuildOptions): MavenProject =
+    MavenProject(
+      settings = Nil
+    )
+
+  private def javacOptionsSettings(options: BuildOptions): List[String] = {
+
+    val javacOptionsSettings =
+      if (options.javaOptions.javacOptions.toSeq.isEmpty) Nil
+      else {
+        val options0 = options
+          .javaOptions
+          .javacOptions
+          .toSeq
+          .map(_.value)
+          .map(o => "\"" + o.replace("\"", "\\\"") + "\"")
+        options0
+      }
+
+    // MavenProject(
+    //   settings = Seq(javacOptionsSettings)
+    // )
+    javacOptionsSettings.toList
+  }
+
+  private def projectArtifactSettings(): MavenProject =
+    // todo: Is it really needed to configure the values, or always use the default?
+    MavenProject(
+      groupId = None,
+      artifactId = None,
+      version = None
+    )
+
+  private def dependencySettings(
+    options: BuildOptions,
+    scope: Scope,
+    sources: Sources
+  ): MavenProject = {
+
+    val scalaV = getScalaVersion(options)
+    def getScalaMajorPrefix =
+      scalaV match {
+        case s"2.12.${patch}" => "2.12"
+        case s"2.13.${patch}" => "2.13"
+        case s"3.$x.$y"       => "3"
+      }
+
+    val depSettings = {
+      def toDependencies(
+        deps: ShadowingSeq[Positioned[AnyDependency]],
+        isCompileOnly: Boolean
+      ): Seq[MavenLibraryDependency] = {
+        val providedDeps = deps.toSeq.toList.map(_.value).map { dep =>
+          val org  = dep.organization
+          val name = dep.name
+          val ver  = dep.version
+          // TODO dep.userParams
+          // TODO dep.exclude
+          // TODO dep.attributes
+          val artNameWithPrefix = dep.nameAttributes match {
+            case NoAttributes           => name
+            case s: ScalaNameAttributes => s"${name}_$getScalaMajorPrefix"
+          }
+          val scope0 =
+            // FIXME This ignores the isCompileOnly when scope == Scope.Test
+            if (scope == Scope.Test) Some("test")
+            else if (isCompileOnly) Some("provided")
+            else None
```

Comment:
as discussed, dependencies in the test scope can't be provided for Maven, it seems.